sources/packages/web/src/components/common/DesignationAgreement/DesignationAgreementForm.models.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Used do define conditions to change the form.io, for instance,
+ * some components visibility.
+ */
+export enum DesignationFormViewModes {
+  submission = "submission",
+  viewOnly = "viewOnly",
```

Comment:
does it mean ReadOnly?

Reply:
Kind off 😄 
I did that on purpose because when the form goes to readOnly/viewOnly it receives new items in the UI, like the status on the header and a new column for approval status. So, since it it was not just about making the components readOnly I try to use a different name.

Reply:
Makes sense.